rock_paper_scissors.py

```diff
@@ -0,0 +1,61 @@
+#!/bin/python3
+print('''Let\'s play a game.
```

Comment:
using multiline strings in this situation makes the code look a bit messy - stylistically I'd prefer either two print statements or adding `\n` to the end of a one line string (e.g. `print("Let's play a game. \n")`.

Note that both `'` and `"` can be used to quote a string, and if you use `"` you don't need to escape `'`.